src/InputWithLabel.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+function InputWithLabel(props) {
+
+  const inputRef = React.useRef();
+  console.log(inputRef.current)
+
+  React.useEffect(() => {
+    inputRef.current.focus();
+  });
+
+  return (
+    <>
+      <label htmlFor="todoTitle">{props.children}</label>
+      <input
+        ref={inputRef}
+        id="todoTitle"
```

Comment:
to make this component fully reusable you can send `todoTitle` and `text` via props too.